core/src/test/java/org/hibernate/ogm/utils/test/SkipByDatastoreProviderSelfJpaTest.java

```diff
@@ -6,14 +6,14 @@
  */
 package org.hibernate.ogm.utils.test;
 
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.CASSANDRA_EXPERIMENTAL;
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.COUCHDB_EXPERIMENTAL;
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.INFINISPAN;
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.MAP;
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.EHCACHE;
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.REDIS_EXPERIMENTAL;
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.NEO4J_EMBEDDED;
-import static org.hibernate.ogm.datastore.impl.AvailableDatastoreProvider.MONGODB;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.CASSANDRA_EXPERIMENTAL;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.COUCHDB_EXPERIMENTAL;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.INFINISPAN;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.MAP;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.EHCACHE;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.REDIS_EXPERIMENTAL;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.NEO4J_EMBEDDED;
+import static org.hibernate.ogm.datastore.impl.DatastoreProviderType.MONGODB;
```

Comment:
I wanted it to be consistent with `GridDialectType` as they are exactly the same thing.